### PR TITLE
docs: fix path to logo

### PR DESCRIPTION
**Pull request by GlennChia**
## Overview

Change path to `tcat.png` to render png logo

Note: The URL https://taskcat.io/ is currently not available

## Testing/Steps taken to ensure quality

Manual verification of README in branch

### CODE REVIEW

**Comment by GlennChia** [comment]
@tonynv, I noticed you just made a fresh PR for something similar. There are 2 READMEs that have the old broken logo link. I've modified my PR to use the link that you used in your PR for both READMEs. Requesting your review for this. Thanks!

**Review by andrew-glenn — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-[![logo](https://aws-ia.github.io/assets/img/taskcat-bg.png)](https://aws-ia.github.io/)
-__https://aws-ia.github.io__
+[![logo](https://raw.githubusercontent.com/aws-quickstart/taskcat/main/assets/docs/images/tcat.png)](https://taskcat.io/)
```
Comment (tonynv) [inline README.md:1]:
```suggestion
[![logo](https://raw.githubusercontent.com/aws-ia/taskcat/main/assets/docs/images/tcat.png)](https://taskcat.io/)
```

File: docs/README.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
-[![logo](https://aws-ia.github.io/assets/img/taskcat-bg.png)](https://aws-ia.github.io/)
-__https://aws-ia.github.io__
+[![logo](https://raw.githubusercontent.com/aws-quickstart/taskcat/main/assets/docs/images/tcat.png)](https://taskcat.io/)
```
Comment (tonynv) [inline docs/README.md:1]:
```suggestion
[![logo](https://raw.githubusercontent.com/aws-ia/taskcat/main/assets/docs/images/tcat.png)](https://taskcat.io/)
```

Comment (tonynv) [inline docs/README.md:1]: We have migrated taskcat to the aws-ia org.  Some changes are needed to the image path in your PR

**Review by tonynv — CHANGES_REQUESTED** [review]

**Review by andrew-glenn — APPROVED** [review]

**Comment by tonynv** [comment]
Please resolve conflicts

**Review by tonynv — APPROVED** [review]

**Comment by GlennChia** [comment]
Thanks @andrew-glenn for resolving the conflicts!

**Review by andrew-glenn — APPROVED** [review]

**Pull request merged**